pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1405,6 +1405,9 @@ self: super: {
   # $HOME, which we don't have in our build sandbox.
   cabal-install-parsers = dontCheck super.cabal-install-parsers;
 
+  # gitit is unbroken in the latest release
+  gitit = markUnbroken super.gitit;
```

Comment:
If you submit this PR to the `haskell-updates` branch, then you shouldn't need this.

As soon as the daily update CI thing runs, `gitit` will be unmarked broken.

Reply:
Is the workings of that written down somewhere?

Reply:
@Profpatsch Unfortunately, not as far as I know.

I wrote about it a little in this post:

https://discourse.nixos.org/t/haskellpackages-stm-containers-fails-to-build/5416/4

Also, @maralorn recently created a blog post that included a short section explaining how this works:

https://mpickering.github.io/ide/posts/2020-06-05-ghcide-and-nixpkgs.html

Reply:
I fear I didn't mention the fact that hackage2nix generates the broken flags explicitly.

Reply:
I am still none the wiser. Is there a step-by-step description of what I have to change where and commit to which branch in what order? 

I will merge this one here for now, but am open to doing it the “correct” way if somebody points me to clear documentation that is not spread over multiple Youtube videos and blogposts.

Reply:
@Profpatsch I completely understand your frustration. But for this narrow use case it's actually really simple:

1. Confirm that package build succeeds, when broken flag is overriden.
2. Make a PR against the `haskell-updates` branch, where you remove the package from the array of broken packages in `configuration-hackage2nix.yaml`.
3. After merge wait for the daily `hackage-packages.nix` update, which removes the broken flag from the package and removes the flag which disables builds on hydra.
4. Every friday `haskell-updates` get's merged into master.

Bonus: You can do step 3. on your own in your merge request, but yeah, I think the documentation for that is a youtube video right now.

We really need to make sure all of this is easily discoverable documented.